MultiVersions/Forge/main/beachparty/lang/zh_tw.json

```
@@ -0,0 +1,143 @@
+{
+  "beachparty.container.cabinet": "櫥櫃",
+  "block.beachparty.beach_chair":"沙灘椅",
+  "block.beachparty.beach_grass": "沙灘草",
+  "block.beachparty.beach_towel": "沙灘巾",
+  "block.beachparty.cabinet": "熱帶風格櫥櫃",
+  "block.beachparty.chair":"棕櫚木椅",
+  "block.beachparty.cocoa_cocktail": "可可豆雞尾酒",
+  "block.beachparty.coconut": "椰子",
+  "block.beachparty.coconut_block": "椰子",
+  "block.beachparty.coconut_cocktail": "椰子雞尾酒",
+  "block.beachparty.deck_chair": "棕櫚木躺椅",
+  "block.beachparty.dried_wheat_block": "乾小麥捆",
+  "block.beachparty.dried_wheat_slab": "乾小麥捆半磚",
+  "block.beachparty.dried_wheat_stairs": "乾小麥捆階梯",
+  "block.beachparty.dry_bush":"乾灌木",
+  "block.beachparty.dry_bush_tall": "高乾灌木",
+  "block.beachparty.empty_sand_bucket": "空沙桶",
+  "block.beachparty.empty_sand_bucket_block": "空沙桶",
+  "block.beachparty.hammock": "沙灘吊床",
+  "block.beachparty.honey_cocktail": "蜂蜜雞尾酒",
+  "block.beachparty.lounge_chair":"棕櫚木躺椅",
```

雖然功能上不能摺疊，但至少外觀上看起來可以 XD
跟 `Deck Chair` 的翻譯作區分
```suggestion
  "block.beachparty.lounge_chair":"棕櫚木折疊椅",
```